README.md

```diff
@@ -28,8 +30,8 @@
 
 The UI for [Spline project](https://absaoss.github.io/spline/)
 
-Spline UI app is implemented as a _single-page application (SPA)_, available alone as well as packed as a Docker image or a WAR-file.
-Choose at your deployment preferences.
+Spline UI app is implemented as a _single-page application (SPA)_, available alone as well as packed as a Docker image or a WAR-file. Choose at your
+deployment preferences.
```

Comment:
```suggestion
Spline UI app is implemented as a _single-page application (SPA)_, available alone as well as packed as a Docker image or a WAR-file. Choose according to your
deployment preferences.
```
IMHO, we could just remove the last sentence. There is now additional information.